db/system_keyspace.cc

```diff
@@ -543,6 +542,46 @@ static schema_ptr large_cells() {
     return scylla_local;
 }
 
+/** Layout based on C*-4.0.0 with extra columns `shard_id' and `client_type'
+  * but without `request_count'. Also CK is different: C* has only (`port'). */
+static schema_ptr clients() {
+    thread_local auto clients = [] {
+        schema_builder builder(make_lw_shared(schema(generate_legacy_id(NAME, CLIENTS), NAME, CLIENTS,
+            // partition key
+            {{"address", inet_addr_type}},
+            // clustering key
+            {{"client_type", utf8_type}, {"port", int32_type}},
+            // regular columns
```

Comment:
Why am I seeing this twice?

Reply:
If `address` and `port` come from cassandra, we should keep the names.

Reply:
Yes, column names come from C*, so I left them as is.